File: app/src/main/java/mb00/android/codehub/ui/home/inject/HomeFollowingFragmentComponent.java
package mb00.android.codehub.ui.home.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.user.view.HomeFollowingFragment;


@Subcomponent(modules = HomeFollowingFragmentModule.class)
public interface HomeFollowingFragmentComponent extends AndroidInjector<HomeFollowingFragment> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<HomeFollowingFragment> {}

}

File: app/src/main/java/mb00/android/codehub/ui/home/inject/HomeFollowingFragmentComponent.kt
package mb00.android.codehub.ui.home.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.user.view.HomeFollowingFragment


@Subcomponent(modules = [
    HomeFollowingFragmentModule::class
])
interface HomeFollowingFragmentComponent : AndroidInjector<HomeFollowingFragment> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<HomeFollowingFragment>()

}
